### PR TITLE
Fix broken About button in commercial components

### PR DESCRIPTION
**Pull request by regiskuckaertz**
The initial call to `Toggles` does not account for lazy loaded components, thus we must call it again in case there's a popup in the content returned from `/commercial`

@jbreckmckye @uplne

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js [hunk 1]
```diff
@@ -93,10 +95,18 @@ define([
             };
         };
 
+    function createToggle(el) {
+        if (el.querySelector('.popup__toggle')) {
+            new Toggles(el).init();
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: Does this component short circuit if you try initializing a toggle twice?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: Yes, this is taken care of in the `Toggles` module.

File: commercial/app/views/debugger/allcreatives.scala.html [hunk 2]
```diff
@@ -645,7 +645,9 @@ <h2 class="test-page-head">
                                 omnitureId: '[%OmnitureID%]'
                             });
 
-                            commercialComponent.postLoadEvents['capi@if(component("type") == "single"){Single}'] = function () {
+                            var oldFn = commercialComponent.postLoadEvents['capi@if(component("type") == "single"){Single}'];
```
Comment (jbreckmckye) [inline commercial/app/views/debugger/allcreatives.scala.html]: In this context I'm not sure if `oldFn`'s meaning is immediately obvious, at least until you read a little further. What about `originalPostLoadEvents`?

**Comment by regiskuckaertz** [comment]
@jbreckmckye thanks, I had the same feeling too :)

**Comment by jbreckmckye** [comment]
Ready to go :+1:

**Pull request merged**